src/base/memory_monitor_sr.cpp

```diff
@@ -20,18 +20,62 @@
  * memory_monitor_sr.cpp - Implementation of memory monitor module
  */
 
+#include <malloc.h>
+#include <cassert>
 #include <cstring>
 #include <algorithm>
-#include <cassert>
 
 #include "memory_monitor_sr.hpp"
 
+#ifndef HAVE_USR_INCLUDE_MALLOC_H
+#define HAVE_USR_INCLUDE_MALLOC_H
+#endif
+
+typedef struct mmon_metainfo
+{
+  uint64_t size;
```

Comment:
NIT:

`alloced_size` is more helpful when I read the code.
```suggestion
  uint64_t alloced_size;
```

Reply:
I think `alloc_size` or `allocated_size` is better because "alloc" is an abbreviation for "allocate", so "alloced" is not appropriate. (I think it should be "alloc-ed.")

Reply:
I change it to `allocated`.